api/proto/teleport/legacy/types/types.proto

```diff
@@ -2939,7 +2939,7 @@ message RoleOptions {
     (gogoproto.customtype) = "BoolOption"
   ];
 
-  // CreateHostUser allows users to be automatically created on a host
+  // Deprecated: Should use CreateHostUserMode instead.
```

Comment:
This comment unfortunately does not show up in the generated terraform documentation. After some digging I learned it's because custom types (e.g. `BoolOption`) don't propagate comments as an attribute description. There's a couple of ways I think we can address this and I'll be opening a PR against the `protoc-gen-terraform` repo shortly with one potential solution.

Reply:
The `protoc-gen-terraform` PR landed and this shows up correctly now :+1: